src/doc/reference.md

```diff
@@ -76,8 +76,12 @@ An identifier is any nonempty Unicode[^non_ascii_idents] string of the following
 [^non_ascii_idents]: Non-ASCII characters in identifiers are currently feature
   gated. This is expected to improve soon.
 
-- The first character has property `XID_start`
-- The remaining characters have property `XID_continue`
+Either
+   * The first character has property `XID_start`
+   * The remaining characters have property `XID_continue`
+Or
+   * The first character is `_`
+   * The remaining characters have property `XID_continue`
```

Comment:
It is not. Should I be more explicit that there must be more than zero 'remaining characters' here?